apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp

```diff
@@ -409,7 +417,10 @@ static void utc_media_MediaPlayer_setVolume_n(void)
 
 int utc_media_MediaPlayer_main(void)
 {
-	SetUp();
+	if (SetUp() != 0) {
```

Comment:
I think let it work will be better than return -1 because in this case tc result will treat it as all succedded

Reply:
In case of failure, do not proceed tc.

Reply:
No, TC_ASSERT increase fail count and TC_SUCCESS_RESULT will increase success count.
So if we do not test all, fail count is not increased.
This doesn't give correct info to user.

Reply:
you right. so i remove return type and just will add notice